### ISSUE TITLE
plot_hemispheres() crashes Google Colab

### ISSUE THREAD

**Issue by Neurojedi** [MICA-MNI/BrainSpace#80]
Hi everyone,

I am trying to use BrainSpace on Google Colab. However, Colab is getting crashed whenever I run plot_hemispheres() function. I tried the tutorial codes provided on https://brainspace.readthedocs.io/en/latest/python_doc/auto_examples/index.html 

None of them is working on Colab, whereas they work on my local machine. Is there a way around solving this problem?

You can also access my code here: https://colab.research.google.com/drive/11RDqIxQQp0i6QMMx8L67C9G3IJ5yplen?usp=sharing

Kind regards.

**Comment by PeerHerholz** [comment]
Hi @Neurojedi,

thx for bringing this up. I think this might be related to `vtk` and some required underlying `libraries` (e.g. `libgl`).
Thus, you might need to run it within a computing environment that allows you to install the respective requirements (e.g. set up a `Dockerfile` and run it via `mybinder`).

Looping in @OualidBenkarim as well.

**Comment by ReinderVosDeWael** [comment]
This sounds analogous to issue #66. If you can get Conda to work inside colab then #66 may offer a solution. If not, then unless Oualid has a solution, I don't think this can be resolved in colab.

**Comment by PeerHerholz** [comment]
Thx for the pointer @ReinderVosDeWael!

**Comment by ReinderVosDeWael** [comment]
I got curious and gave the solution of #66 a try in colab. Unfortunately it does not work.

**Comment by OualidBenkarim** [comment]
Thanks guys!

This is definitely related to vtk. I will check it out in more depth

**Comment by Neurojedi** [comment]
Hello everyone,

I am happy to report that yesterday I was able to successfully use plot_hemispheres() on Colab using the instructions provided in issue #66. Previously, I had encountered issues while trying to do the same thing, but this time it worked without fail. I think the reason for the success is that I used 'condacolab' to install Conda on Colab, whereas I had previously used Miniconda, which may have caused compatibility problems. For those who are interested, I have provided the steps that I used for installation here: https://colab.research.google.com/drive/1YLqOk0uRiccqRRp5zb2bwu5utQiV8mlE?usp=sharing
